parley/src/style/mod.rs

```diff
@@ -22,6 +22,48 @@ pub enum WhiteSpaceCollapse {
     Preserve,
 }
 
+/// Control over where words can wrap.
+///
+/// See <https://drafts.csswg.org/css-text/#propdef-word-break> for more information.
+#[derive(Copy, Clone, Default, PartialEq, Eq, Debug)]
+#[repr(u8)]
+pub enum WordBreak {
+    /// Words can be broken according to their normal Unicode rules.
+    #[default]
+    Normal,
+    /// Breaking is allowed within words. This does not affect breaking around punctuation.
+    BreakAll,
+    /// Breaking is forbidden within words.
+    KeepAll,
+}
+
+impl WordBreak {
+    pub(crate) fn as_swash(&self) -> swash::text::WordBreakStrength {
+        match self {
+            Self::Normal => swash::text::WordBreakStrength::Normal,
+            Self::BreakAll => swash::text::WordBreakStrength::BreakAll,
+            Self::KeepAll => swash::text::WordBreakStrength::KeepAll,
+        }
+    }
+}
```

Comment:
I think it makes sense to add the `default` to `WordBreakStrength` in Swash and reuse/reexport that enum here.

Reply:
I'm a bit apprehensive to pass through more data types directly from Swash since the long-term plan is to move away from it*, but I suppose this enum isn't doing much right now, so I've done so.

*Also, I would like to add serde support to this crate at some point, and adding it to Swash as well would be a bit of an ordeal

Reply:
I see, that is a valid reason.

Reply:
It would definitely be nice if we could get to the point where Swash wasn't in Parley's public API such that we could update Swash without it being a semver-breaking change for Parley. Not sure how far away from that we are atm.